frontend/pages/home/style.scss

```diff
@@ -1 +1,131 @@
 @import "../../App.scss";
+
+.date-picker {
+  width: 60%;
+  fieldset {
+    border-color: #68658b !important;
+  }
+  input {
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+    font-size: 0.75rem;
+    color: #ffffff;
```

Comment:
can you use tailwind colors? or maybe a global variable? is not flexible in case there is an update on the colors of the UI.

Reply:
Done